openncp-gateway-backend/manifests/openncp-gateway-backend-deployment.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: openncp-gateway-backend
+  namespace: openncp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: openncp-gateway-backend
+  template:
+    metadata:
+      labels:
+        app: openncp-gateway-backend
+    spec:
+      containers:
+      - name: openncp-gateway-backend
+        image: docker.io/3ahealth/openncp-gateway-backend:7.1.0
+        ports:
+        - containerPort: 11080
+        - containerPort: 11443
+        resources:
+          requests:
+            memory: "512Mi"
+            cpu: "500m"
+          limits:
+            memory: "1Gi"
+            cpu: "1"
+        env:
+        - name: EPSOS_PROPS_PATH
+          value: "/opt/epsos-configuration/"
+        - name: CATALINA_OPTS
+          value: "
+            -DopenATNA.properties.path=file:/opt/epsos-configuration/ATNA_resources/openatna.properties
+            -Dfile.encoding=UTF-8"
+        - name: MYSQL_HOST
+          value: "mysql-service"
+        - name: MYSQL_PORT
+          value: "3306"
+        - name: MYSQL_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: mysql-secret
+              key: MYSQL_USER
+        - name: MYSQL_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: mysql-secret
+              key: MYSQL_PASSWORD
+        - name: TLS_KEYSTORE_PATH
```

Comment:
Please add instructions in the README how to generate the jks files

Reply:
Working on it! This will be added in a next PR where I will define more information in the readme!